src/zenml/zen_stores/migrations/versions/a91762e6be36_artifact_version_table.py

```diff
@@ -0,0 +1,228 @@
+"""Artifact Version Table [a91762e6be36].
```

Comment:
I would really recommend testing this against different scenarios as it is a significant migration IMO.

Reply:
Yeah agreed, so far I tested the following scenarios:
- SQLite initialization from scratch
- SQLite upgrade from 0.50.0 with existing artifacts
- MySQL initialization from scratch
- MySQL initialization from 0.50.0 with existing artifacts

Any other case I should test?

Reply:
In general, I can't think of a different test. 

However, In my experience, the migration failures mostly stem from (sort of) edge cases on various versions way before 0.50.0 or so. If there is such a case, (for instance where a specific field of the artifact can be possibly null), I would try to recreate this scenario and test that as well.